.env

```diff
@@ -16,7 +16,7 @@ ENV_GORELEASER_VERSION=v1.23.0
 
 ## Default Docker Images
 DEFAULT_FLUENTBIT_EXPORTER_IMAGE="europe-docker.pkg.dev/kyma-project/prod/directory-size-exporter:v20241212-e4adf27f"
-DEFAULT_FLUENTBIT_IMAGE="europe-docker.pkg.dev/kyma-project/prod/external/fluent/fluent-bit:3.2.2"
+DEFAULT_FLUENTBIT_IMAGE="europe-docker.pkg.dev/kyma-project/prod/external/fluent/fluent-bit:3.1.9"
```

Comment:
Why you are reverting the fluentbit version? To get new numbers for that version it is not required to change the version on main